basic/.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ cache:
     - cache/Cypress
 
 test:
-  image: cypress/base:10
```

Comment:
Cypress v12 dropped node 12 support. This is an example for users. Updated to pull latest.